dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/dialogEditBindPrimaryDomain.xml

```diff
@@ -23,6 +23,11 @@
         <type>select_multiple</type>
         <help>Define the ACLs where you allow which client are allowed to query this zone.</help>
     </field>
+    <id>domain.allowrndcupdate</id>  
+        <label>Allow rndc-key update</label> 
+        <type>checkbox</type>        
+        <help>By ticking this, you enable updates via the rndc-key named "rndc-key". The key is generated in the general tab</help>
```

Comment:
Updated the text to be somewhat shorter and more clear.

Reply:
Better already. But I meant the rndc-key part occurring twice specifically. Sorry for not being clear.

Reply:
Aha, ok. Well, it's a bit of a redundancy since the keys actual name is "rndc-key" and there is no implemented way of changing it. To use the key, you need to add both the value of the key, and the name, on the downstream dns server